Runtime/Scripts/msCoreAPI.cs

```diff
@@ -2950,8 +3034,74 @@ public Matrix4x4[] transforms
 
 #endregion
 
-internal struct SceneData {
-    #region internal
+
+
+#region PropertyInfo
+
+    public enum PropertyInfoDataType
```

Comment:
Failed to notice this again...
Can you make this to internal ?

Reply:
Done